### PR TITLE
7.0 add Web Custom Login and Web Login Background

### PR DESCRIPTION
**Pull request by ntodorova**
Those two modules allow to customize your log in page.
Web Custom Login
- allow to set image different from your logo as a login image.

Web Login Background
- allow to set login background image
- allow to hide the login image

### CODE REVIEW

File: web_custom_login/__init__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline web_custom_login/__init__.py:1]: @ntodorova   Thanks for your contribution to the OCA. 
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code.
In general, please follow the principles: 
- Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
- Please follow the OCA README template:
  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
- We recommend to include a screenshot as well to help new comers (nice to have)
- Please use simple headers in all py files following OCA templates
  https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3
- Please sort all imports in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

Comment (elicoidal) [inline web_custom_login/__init__.py:1]: In particular, init.py files usually dont have copyright notice. for any other py file please use simplified copyright notice.

File: web_custom_login/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "summary":  "Login page depending on selected database",
+    'license':  'AGPL-3',
+    "description": """
+Make login img customizable.
+Login page depending on selected database.
+    """,
+    'data': [
+        'res_company_view.xml'
+    ],
+    'demo': [
```
Comment (elicoidal) [inline web_custom_login/__openerp__.py]: remove empty keys

File: web_login_background/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Web Login Background",
```
Comment (elicoidal) [inline web_login_background/__openerp__.py:5]: Why dont you merge the 2 modules?

File: web_login_background/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    "category": "",
+    'license': 'AGPL-3',
+    "description": """
+Add new login background img.
+Use background img if it is set.
+Allow to remove the login img.
+    """,
+    'data': ["res_company_view.xml",
+             ],
+    'demo': [
```
Comment (elicoidal) [inline web_login_background/__openerp__.py]: remove empty keys?

File: web_login_background/controllers/main.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+            uid = openerp.SUPERUSER_ID
+
+        if not dbname:
+            image_data = self.placeholder(req, 'background.png')
+        else:
+            try:
+                # create an empty registry
+                registry = openerp.modules.registry.Registry(dbname)
+                with registry.cursor() as cr:
+                    cr.execute("""SELECT c.login_background_img
```
Comment (elicoidal) [inline web_login_background/controllers/main.py:34]: What is the reason to use SQL?

File: web_custom_login/controllers/main.py [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+        if not uid:
+            uid = openerp.SUPERUSER_ID
+
+        if not dbname:
+            image_data = self.placeholder(req, 'logo.png')
+        else:
+            try:
+                # create an empty registry
+                registry = openerp.modules.registry.Registry(dbname)
```
Comment (elicoidal) [inline web_custom_login/controllers/main.py:59]: What is the reason to use SQL?

File: web_login_background/controllers/main.py [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+            uid = openerp.SUPERUSER_ID
+
+        if not dbname:
+            display = 'False'
+        else:
+            try:
+                # create an empty registry
+                registry = openerp.modules.registry.Registry(dbname)
+                with registry.cursor() as cr:
+                    cr.execute("""SELECT c.display_login_logo
```
Comment (elicoidal) [inline web_login_background/controllers/main.py:73]: idem

**Comment by elicoidal** [comment]
Test OK. some details about copyright. I miss the README file as well.
Those 2 features are complementary: it would make sense to merge both modules but I wouldnot block the PR for that
Good  contribution!

**Comment by ntodorova** [comment]
@elicoidal Thank you very much for your comments and sorry for the delay!

File: web_login_background/controllers/main.py (hunk 5 above)
Comment (ntodorova) [inline web_login_background/controllers/main.py:34]: Do you have any better suggestion in that case?

**Pull request closed without merging**